sources/launcher/Stride.Launcher/ViewModels/ReleaseNotesViewModel.cs

```diff
@@ -68,26 +70,17 @@ internal ReleaseNotesViewModel([NotNull] LauncherViewModel launcher, [NotNull] s
 
         public async void FetchReleaseNotes()
         {
-            string releaseNotesMarkdown = null;
+            string releaseNotesMarkdown;
 
             try
             {
-                var request = WebRequest.Create($"{BaseUrl}{ReleaseNotesFileName}");
-                using (var response = await request.GetResponseAsync())
+                using (var response = await httpClient.GetAsync($"{BaseUrl}{ReleaseNotesFileName}"))
                 {
-                    using (var str = response.GetResponseStream())
-                    {
-
-                        if (str != null)
-                        {
-                            using (var reader = new StreamReader(str))
-                            {
-                                releaseNotesMarkdown = reader.ReadToEnd();
-                            }
-                        }
-                    }
-                    // fetch the response Uri and update the base URL
```

Comment:
This part seemed a bit odd since it appears that the BaseUrl isn't used again after this.   I left the functionality of the code intact, but I'd suggest that updating the BaseUrl isn't really necessary, even if there were a redirect.   I'm happy to remove this code as a part of this change or as a follow-on if reviewers agree it's not needed.

Reply:
I don't have a strong opinion on that, but if the code is mostly pointless then I have nothing against removing it.

Reply:
Code and set removed in second commit.